lib/configuration.js

```diff
@@ -22,6 +22,18 @@ class Configuration {
         this.settings.mergeable[key] = Configuration.DEFAULTS[key]
       }
     }
+
+    // ensure stale defaults
+    if (this.settings.mergeable.pull_requests &&
+      this.settings.mergeable.pull_requests.stale !== undefined &&
```

Comment:
i think we don't need `!== undefined` here. because we also don't want the `stale` to be null right?

Reply:
well, currently `stale` is undefined if it is not specified in the yml.

Reply:
I see what you're saying. You're right.